frontend/src/pages/Notifications/NotificationList.js

```diff
@@ -100,7 +100,7 @@ const NotificationsList = props => {
               markPageAsRead(markMultipleNotificationsAsRead, notifications, notificationOffset, notificationsPerPage)
             }
             color="primary"
-            className={classes.button}
+            className={classes.button + " mark-all-notifications-as-read"}
```

Comment:
Aren't we using inline-styles instead of classes?

Reply:
yes, you are right. changed it to use only the `data-test` tag in the e2e tests for notifications.